### ISSUE TITLE
Checkbox and Radiobutton widget

### ISSUE THREAD

**Issue by jkvarel** [inventwo/ioBroker.vis-2-widgets-inventwo#4]
See Checkbox/Radiobutton widget in old adapter

**Comment by nesheimt** [comment]
Yes, this would be great if added to the VIS2 widgets. I use the VIS-checkbox/radiobutton widget in VIS2 but they do not work for touch screens.

**Comment by nesheimt** [comment]
Sorry, then I was not precise. I'n using the checkbox list for manually beiung able to adjust heat storage:

<img width="570" height="197" alt="Image" src="https://github.com/user-attachments/assets/87c474ba-1a1a-4e4e-bf98-7f1a90eecf6a" />

Not able to find this in the new widget, but create a large number of individual checkboxes and align them
